actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -514,6 +523,26 @@ def resolve_image_source(source, skip_pipeline)
           raise ArgumentError, "Can't resolve image into URL: #{e}"
         end
 
+        def resize_image_source(source, options)
+          return source unless resize_active_storage_images
+          return source unless source.respond_to?(:variable?) && source.variable?
```

Comment:
The safe navigator was not enough since the source could be a string or a variant, neither of which respond to `variable?`